src/components/Toggle/stories/Toggle.checkbox.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { Toggle } from '../..';
+
+<Meta title="Components/Toggle/Checkbox" component={Toggle} />
+
+export const Template = (args) => <Toggle {...args} />;
+
+# Toggle
+
+## Checkbox Toggle
```

Comment:
What is this checkbox toggle supposed to be? Is it supposed to trigger something on the page?

Reply:
I updated the stories to better explain it to users. Let me know if it is still not clear.